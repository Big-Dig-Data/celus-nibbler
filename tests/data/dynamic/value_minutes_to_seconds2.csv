Title,ISSN,Platform,YOP,Metrics,2020-01,2020-02,2020-03
"T1","1111-1111","P1","2015","Minutes","1","10","60"
"T1","1111-1111","P1","2015","Users","2","4","8"
"T2","2222-2222","P1","2013","Minutes","6","2.5","0.3"
"T2","2222-2222","P1","2013","Users","3","9","27"
