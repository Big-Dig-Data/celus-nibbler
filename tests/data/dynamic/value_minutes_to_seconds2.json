{
  "kind": "non_counter.generic",
  "version": 1,
  "parser_name": "value_minutes_to_seconds2",
  "data_format": {
    "name": "minutes_format"
  },
  "platforms": [
    "Platform1",
    "Platform2"
  ],
  "metrics_to_skip": [],
  "titles_to_skip": [],
  "dimensions_to_skip": {},
  "metric_aliases": [
      ["Minutes", "Seconds"]
  ],
  "metric_value_extraction_overrides": {
    "Seconds": "minutes_to_seconds"
  },
  "dimension_aliases": [],
  "heuristics": {
    "conds": [
      {
        "pattern": "Title",
        "coord": {
          "row": 0,
          "col": 0
        },
        "kind": "regex"
      },
      {
        "pattern": "ISSN",
        "coord": {
          "row": 0,
          "col": 1
        },
        "kind": "regex"
      },
      {
        "pattern": "Platform",
        "coord": {
          "row": 0,
          "col": 2
        },
        "kind": "regex"
      },
      {
        "pattern": "YOP",
        "coord": {
          "row": 0,
          "col": 3
        },
        "kind": "regex"
      },
      {
        "pattern": "Metrics",
        "coord": {
          "row": 0,
          "col": 4
        },
        "kind": "regex"
      }
    ],
    "kind": "and"
  },
  "areas": [
    {
      "data_headers": {
        "roles": [
            {
                "source": {
                    "coord": {
                      "row": 0,
                      "col": 5
                    },
                    "direction": "right"
                },
                "role": "date"
            }
        ],
        "data_direction": "down",
        "data_cells": {
          "coord": {
            "row": 1,
            "col": 5
          },
          "direction": "right"
        },
        "data_extract_params": {
            "special_extraction": "comma_separated_number"
        }
      },
      "titles": {
        "source": {
          "coord": {
            "row": 1,
            "col": 0
          },
          "direction": "down"
        }
      },
      "metrics": {
        "source": {
          "coord": {
            "row": 1,
            "col": 4
          },
          "direction": "down"
        }
      },
      "title_ids": [
        {
          "name": "Print_ISSN",
          "source": {
            "coord": {
              "row": 1,
              "col": 1
            },
            "direction": "down"
          }
        }
      ],
      "dimensions": [
        {
          "name": "YOP",
          "source": {
            "coord": {
              "row": 1,
              "col": 3
            },
            "direction": "down"
          }
        },
        {
          "name": "Platform",
          "source": {
            "coord": {
              "row": 1,
              "col": 2
            },
            "direction": "down"
          }
        }
      ],
      "kind": "non_counter.generic"
    }
  ]
}
