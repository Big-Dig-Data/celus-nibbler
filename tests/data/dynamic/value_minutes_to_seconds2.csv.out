"2020-01-01","2020-01-31","","T1","Seconds","Platform:P1|YOP:2015","Print_ISSN:1111-1111","60"
"2020-02-01","2020-02-29","","T1","Seconds","Platform:P1|YOP:2015","Print_ISSN:1111-1111","600"
"2020-03-01","2020-03-31","","T1","Seconds","Platform:P1|YOP:2015","Print_ISSN:1111-1111","3600"
"2020-01-01","2020-01-31","","T1","Users","Platform:P1|YOP:2015","Print_ISSN:1111-1111","2"
"2020-02-01","2020-02-29","","T1","Users","Platform:P1|YOP:2015","Print_ISSN:1111-1111","4"
"2020-03-01","2020-03-31","","T1","Users","Platform:P1|YOP:2015","Print_ISSN:1111-1111","8"
"2020-01-01","2020-01-31","","T2","Seconds","Platform:P1|YOP:2013","Print_ISSN:2222-2222","360"
"2020-02-01","2020-02-29","","T2","Seconds","Platform:P1|YOP:2013","Print_ISSN:2222-2222","150"
"2020-03-01","2020-03-31","","T2","Seconds","Platform:P1|YOP:2013","Print_ISSN:2222-2222","18"
"2020-01-01","2020-01-31","","T2","Users","Platform:P1|YOP:2013","Print_ISSN:2222-2222","3"
"2020-02-01","2020-02-29","","T2","Users","Platform:P1|YOP:2013","Print_ISSN:2222-2222","9"
"2020-03-01","2020-03-31","","T2","Users","Platform:P1|YOP:2013","Print_ISSN:2222-2222","27"
